include/rovio/PoseUpdate.hpp

```diff
@@ -60,8 +60,10 @@ class PoseUpdateMeas: public LWF::State<LWF::VectorElement<3>,LWF::QuaternionEle
  public:
   static constexpr unsigned int _pos = 0;
   static constexpr unsigned int _att = _pos+1;
+  M3D pos_cov_; // Will be used to scale the update covariance according to the measurement
```

Comment:
Ok, it makes sense. It would be a global toggle + the two existing ones for position and attitude.

I've thought about non-linearities in the innovation covariance inversion
step (`Py_`) and believe that the off-diagonal blocks should be zeroed out
if either position or attitude is set to be unused. This is not a problem
in the current code because `updnoiP_` is diagonal by construction, but it
would become so if we allow arbitrary 6x6 covariance matrices. Do you agree?

In other words, the current code relies on the fact that the inverse of a
block-diagonal matrix is still a block-diagonal matrix and the inversion
can be done blockwise. This is not true in the general case we want to
allow here ([reference](
https://en.wikipedia.org/wiki/Block_matrix#Block_matrix_inversion)).